crates/swc_ecma_minifier/tests/benches-full/lodash.js

```diff
@@ -1504,8 +1504,8 @@
             var func = Math[methodName];
             return function(number, precision) {
                 if (number = toNumber(number), (precision = null == precision ? 0 : nativeMin(toInteger(precision), 292)) && nativeIsFinite(number)) {
-                    var pair = (toString(number) + 'e').split('e'), value = func(pair[0] + 'e' + (+pair[1] + precision));
```

Comment:
I think this can be problematic if `pair[0]` modifies `toString`

Reply:
It's the goal of infection analysis (although I'm not sure if this is a correct name)

Reply:
I've tested this code in terser and it generates identical output.

> I think this can be problematic if pair[0] modifies toString

Do you mean access to pair[0] may modify toString?

Reply:
Yes, we may need more execution tests with getters with side effects. e.g.

```js

let foo = () => 1;

const obj = {
    get 0() {
        foo = () => 2
        return 40
    }
};

```

Reply:
Oh, I have an idea. I'll review this by diffing with terser

Reply:
Seems like this to be the same reason as terser/terser/#1244

Reply:
Oh... I see. Thank you!

Reply:
In this case
```js
let foo = ()=>1;
const obj = {
    get 0 () {
        return foo = ()=>2, 40;
    }
};
var c = obj(0);
console.log(foo(c));
```
this PR would generate
```js
let foo = ()=>1;
const obj = {
    get 0 () {
        return foo = ()=>2, 40;
    }
};
var c = obj(0);
console.log(foo(c));
```
So I guess it's fine.

Reply:
It's not the exact problematic case. We can't know input.
Sometimes there are even already-minified code in the input, which is hard to imagine

Reply:
Oh, wait, this is wrong, let me recheck it.

Reply:
So, for code
```js
let foo = () => 1;

const obj = {
    get 0() {
        foo = () => 2;
        return 40;
    },
};
console.log(obj)

var c = obj[0];

console.log(foo(c));
```
SWC would incorrectly inline c because of #5961. I believe the solution is very simple -- any reassignment to foo should prevent seq inline. However this has nothing to do with this PR.

Reply:
Yeah ignoring side effects of getters is fine, as terser does it too

Reply:
Problem is the case below

Reply:
No, terser wouldn't inline `c` here.

Reply:
Oh, so infection analysis is not catching the related of `obj` and `foo`.
Maybe I need to file an issue...

Reply:
I don't think it has any relation with infection analysis in this case. Any reassignment to `foo` should prevent seq inline.

Reply:
Oh yes it would be better